lib/cli/commands-schema/aws-service.js

```diff
@@ -36,16 +36,7 @@ commands.set('deploy', {
     },
   },
   // TODO: Remove deprecated events with v3
```

Comment:
Might be worth removing the TODO comment too?

Reply:
Thanks, good find. Fixed